Documentation/ApiOverview/Events/Events/Redirects/SlugRedirectChangeItemCreatedEvent.rst

```diff
@@ -81,6 +81,32 @@ The corresponding event listener class:
         }
     }
 
+Example of a :php:`CustomSource` implementation:
+
+..  code-block:: php
```

Comment:
this should go into its own file

Reply:
Hmm, then we should move all event examples into separate files.

Reply:
In the long run I think that would be easier to keep up to date. But for now lets keep it like that

Reply:
Yup, will put it on my list.